src/components/side_panel/chart/line_chart/line_chart_config_panel.ts

```diff
@@ -31,6 +31,7 @@ export class LineConfigPanel extends LineBarPieConfigPanel {
         value: (this.props.definition as LineChartDefinition).labelsAsText,
         label: _t("Treat labels as text"),
         onChange: this.onUpdateLabelsAsText.bind(this),
+        disabled: false,
```

Comment:
remove this. It's optional

Reply:
Typescript gives me an error if a remove it. But I'll try to find another way to fix this error then

Reply:
Git it, I needed to add the typing on the super definition